lezgoo.ftd

```diff
@@ -0,0 +1,114 @@
+-- import: fifthtry.github.io/admint.io/assets
+-- import: fifthtry.github.io/admint-cs as cs
+-- import: fifthtry.github.io/admint-typography as typo
+
+
+
+
+
+-- ftd.column:
+width: fill-container
+colors: $cs.main
+types: $typo.types
+
+
+-- lezgoo: Confused about everything Web 3.0? 
+cta-label: Let's Learn
+cta-url: /
+image: $assets.files.static.lezgoo.lezgoo-img.svg
+body1: Don't 
```

Comment:
@SuryanshShahi`body1` is not required here.
Check:
https://admint.io/#/
"Confused about everything Web 3.0? Don't
worry, we will help you ! Lezzzgoooooo!"

Why do you want to split the line?